cmd/terramate/e2etests/runner_unix_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 Mineiros GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build aix || android || darwin || dragonfly || freebsd || hurd || illumos || ios || linux || netbsd || openbsd || solaris
+
+package e2etest
+
+import (
+	"os"
+	"syscall"
+
+	"github.com/madlambda/spells/assert"
+)
+
+// build flags for unix below were taken from:
+// https://github.com/golang/go/blob/master/src/cmd/dist/build.go#L943
+// We can replace them by "unix" if we support only go >= 1.19 later.
```

Comment:
@katcipis oh gosh, the gofmt moved the `// go:build` line to the top and now the comment is orphaned haha I will fix.